plugins/coverbrowser.koplugin/main.lua

```diff
@@ -266,6 +269,50 @@ function CoverBrowser:addToMainMenu(menu_items)
                 },
                 separator = true,
             },
+            {
+                text = _("Series display mode"),
```

Comment:
I'd rather see it in/as `Other settings > Series display mode >` (a bit _niche_ to be in the 1st level).
(With the 2 `Append series metadata...` in it, and even with your `(Detailed list) Files per page` added, we'll have a `Other settings >` with still and only 10 settings, that will fit on one page.

Reply:
done